fast-dump/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+dependencies {
+    implementation deps.kotlin.stdlib
+    implementation deps.androidx.annotation
+}
+
+android {
+    compileSdkVersion versions.compileSdk
+    defaultConfig {
+        minSdkVersion versions.minSdk
+
+        ndkVersion '21.4.7075529'
```

Comment:
This should probably be extracted to the dependencies variable.

Reply:
done